x-pack/plugins/security_solution/public/common/components/event_details/table/add_to_timeline_cell.test.tsx

```diff
@@ -58,7 +58,7 @@ describe('AddToTimelineCellRenderer', () => {
           />
         </TestProviders>
       );
-      expect(screen.getByTestId('test-add-to-timeline')).toBeInTheDocument();
+      expect(screen.queryByLabelText('Investigate in timeline')).toBeInTheDocument();
```

Comment:
Since `getByTestId` can't be used here, consider adding the same

```ts
import { ACTION_INVESTIGATE_IN_TIMELINE } from '../../../../detections/components/alerts_table/translations';
```

as the component, and updating this (and the other assertion below) to use the imported constant, i.e.

```ts
expect(screen.queryByLabelText(ACTION_INVESTIGATE_IN_TIMELINE)).toBeInTheDocument();
```